ports/windows/Makefile

```diff
@@ -49,6 +50,9 @@ ifeq ($(MICROPY_USE_READLINE),1)
 CFLAGS_MOD += -DMICROPY_USE_READLINE=1
 SRC_C += shared/readline/readline.c
 endif
+# ifeq ($(MICROPY_PY_THREAD),1)
```

Comment:
I'd think MicroPython's policy is to not check in dead code?

Reply:
Whoops yeah, thanks for that. I initially expected to need `MICROPY_PY_THREAD` to be defined in makefile snippet like other ports, ended up not turning out that way as it doesn't need extra linker argurments. 

I did push this branch along with the other windows ones I'm working on in a rush before the meetup so didn't do a personal git review properly... sorry for the extra noise :-/